PendingReleaseNotes

```diff
@@ -57,6 +57,14 @@
   default).
 * The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard 'AT_STATX_DONT_SYNC'
   macro. The 'AT_NO_ATTR_SYNC' macro will be removed in the future.
+* Trimming of PGLog dups is now controlled by the size instead of the version.
```

Comment:
how about "is now based on the number of dup entries, instead of versions, and is bound by osd_pg_log_dups_tracked"? 

We should also update https://github.com/ceph/ceph/blob/main/src/common/options/global.yaml.in#L2993-L2995